internal/v4/api_test.go

```diff
@@ -668,6 +668,14 @@ func (s *APISuite) TestMetaPerm(c *gc.C) {
 			Read:  []string{"charmers"},
 			Write: []string{"charmers"},
 		},
+		params.BetaChannel: {
```

Comment:
This is getting really quite repetitive I wonder if some kind of helper function might make things a little more concise.